components/template/Templates.vue

```diff
@@ -1,47 +1,63 @@
 <template>
   <div>
-    <Hero @searchInput="handleSearch" />
-    <div class="template_container d-flex">
+    <div class="container template-container d-flex">
       <div class="left-sidebar">
         <TemplateCategories
           :activeCategory="activeCategory"
           :templateCategories="templateCategories"
         />
       </div>
-      <div class="w-100">
-        <h2 class="heading">
-          {{
-            $route.params.slug ? $route.params.slug.replace('-', ' ') : 'All'
-          }}
+      <div class="content-wrapper w-100">
+        <div
+          v-if="activeCategory"
+          class="breadcrumb d-flex align-items-center gap-2"
+        >
+          <NuxtLink to="/templates/" class="breadcrumb-text">
+            All Templates
+          </NuxtLink>
+          <nuxt-img src="icons/chevron-right.svg" />
+          <span class="breadcrumb-text">{{ activeCategory.name }}</span>
+        </div>
+        <h1 class="content-heading mt-2">
+          {{ activeCategory ? activeCategory.name : 'All' }}
           Templates
-        </h2>
-        <div v-if="filterTemplates && filterTemplates.length" class="templates">
+        </h1>
+        <!-- <p class="content-description">
+          {{ activeCategory.description }}
+        </p> -->
+
+        <TemplateSearch @searchInput="handleSearch" />
```

Comment:
Since now we have description for the templates. I believe the search should also respect that. If the search term matches any word in description we should show that template too.

Reply:
we already had the description for templates. We are adding descriptions for categories.

Reply:
You are right, although we had a description we are only searching through the name.

Reply:
I have implemented it.